LEAF_Request_Portal/js/vue_conditions_editor/LEAF_conditions_editor.js

```diff
@@ -164,30 +173,26 @@ const ConditionsEditor = Vue.createApp({
                 this.selectableParents = this.indicators.filter(i => {
                     return parseInt(i.headerIndicatorID) === headerIndicatorID && 
                             parseInt(i.indicatorID) !== parseInt(this.childIndicator.indicatorID) &&
-                            i.format.indexOf('dropdown') === 0;  //TEST dropdowns and single text only
+                            i.format.indexOf('dropdown') === 0;  //parents are currently dropdowns only
                 });
-                /*if(indicator.conditions !== null && indicator.conditions !== ''){
-                    const conditionObj = JSON.parse(indicator.conditions);
-                    if(conditionObj.length===1){
-                        this.selectConditionFromList(conditionObj[0]);    
-                    }
-                }*/
             }
-            const xhttpForm = new XMLHttpRequest();
-            xhttpForm.onreadystatechange = () => {
-                if (xhttpForm.readyState == 4 && xhttpForm.status == 200) {
-                    const form = JSON.parse(xhttpForm.responseText);
+            $.ajax({

```

Comment:
Was there a reason for switching to the jquery ajax vs native ajax or axios?

Reply:
Mostly I want to remain consistent until a change is decided upon.  Will bring it up again at the meeting.